src/common/types.ts

```diff
@@ -2,9 +2,9 @@
 
 import { ComponentType, ReactNode } from 'react';
 
-import { History, Location as LocationShape } from 'history';
+import { History, Location as HistoryLocationShape } from 'history';
 
-export { LocationShape };
+export type LocationShape = HistoryLocationShape;
```

Comment:
I assume export type here is fine because we export `LocationShape` (without `type`) from index, right?

Reply:
This change fixed the `Attempted import error` with CRA. For some reason, tsc compiled output was not stripping out the `LocationShape` type from `types.js` after compilation. This change got it working right.